### ISSUE TITLE
2025-05-15, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#170]
# Today's Arxiv Papers

(1) [Customizing a Large Language Model for VHDL Design of High-Performance Microprocessors](http://arxiv.org/abs/2505.09610v1), 2025-05-14

(2) [Adversarial Suffix Filtering: a Defense Pipeline for LLMs](http://arxiv.org/abs/2505.09602v1), 2025-05-14

(3) [How Hungry is AI? Benchmarking Energy, Water, and Carbon Footprint of LLM Inference](http://arxiv.org/abs/2505.09598v1), 2025-05-14

(4) [WorldView-Bench: A Benchmark for Evaluating Global Cultural Perspectives in Large Language Models](http://arxiv.org/abs/2505.09595v1), 2025-05-14

(5) [Beyond Likes: How Normative Feedback Complements Engagement Signals on Social Media](http://arxiv.org/abs/2505.09583v1), 2025-05-14

(6) [Ethics and Persuasion in Reinforcement Learning from Human Feedback: A Procedural Rhetorical Approach](http://arxiv.org/abs/2505.09576v1), 2025-05-14

(7) [MIGRATION-BENCH: Repository-Level Code Migration Benchmark from Java 8](http://arxiv.org/abs/2505.09569v1), 2025-05-14

(8) [PT-MoE: An Efficient Finetuning Framework for Integrating Mixture-of-Experts into Prompt Tuning](http://arxiv.org/abs/2505.09519v1), 2025-05-14

(9) [Layered Unlearning for Adversarial Relearning](http://arxiv.org/abs/2505.09500v1), 2025-05-14

(10) [Card Sorting Simulator: Augmenting Design of Logical Information Architectures with Large Language Models](http://arxiv.org/abs/2505.09478v1), 2025-05-14

(11) [Deploying Foundation Model-Enabled Air and Ground Robots in the Field: Challenges and Opportunities](http://arxiv.org/abs/2505.09477v1), 2025-05-14

(12) [Omni-R1: Do You Really Need Audio to Fine-Tune Your Audio LLM?](http://arxiv.org/abs/2505.09439v1), 2025-05-14

(13) [Evaluating GPT- and Reasoning-based Large Language Models on Physics Olympiad Problems: Surpassing Human Performance and Implications for Educational Assessment](http://arxiv.org/abs/2505.09438v1), 2025-05-14

(14) [CXMArena: Unified Dataset to benchmark performance in realistic CXM Scenarios](http://arxiv.org/abs/2505.09436v1), 2025-05-14

(15) [Endo-CLIP: Progressive Self-Supervised Pre-training on Raw Colonoscopy Records](http://arxiv.org/abs/2505.09435v1), 2025-05-14

(16) [SafePath: Conformal Prediction for Safe LLM-Based Autonomous Navigation](http://arxiv.org/abs/2505.09427v1), 2025-05-14

(17) [FaceShield: Explainable Face Anti-Spoofing with Multimodal Large Language Models](http://arxiv.org/abs/2505.09415v1), 2025-05-14

(18) [The Influence of Human-inspired Agentic Sophistication in LLM-driven Strategic Reasoners](http://arxiv.org/abs/2505.09396v1), 2025-05-14

(19) [Quantum-Enhanced Parameter-Efficient Learning for Typhoon Trajectory Forecasting](http://arxiv.org/abs/2505.09395v1), 2025-05-14

(20) [Qwen3 Technical Report](http://arxiv.org/abs/2505.09388v1), 2025-05-14

(21) [MAKE: Multi-Aspect Knowledge-Enhanced Vision-Language Pretraining for Zero-shot Dermatological Assessment](http://arxiv.org/abs/2505.09372v1), 2025-05-14

(22) [Insights into DeepSeek-V3: Scaling Challenges and Reflections on Hardware for AI Architectures](http://arxiv.org/abs/2505.09343v1), 2025-05-14

(23) [RAG-Enabled Intent Reasoning for Application-Network Interaction](http://arxiv.org/abs/2505.09339v1), 2025-05-14

(24) [Llama See, Llama Do: A Mechanistic Perspective on Contextual Entrainment and Distraction in LLMs](http://arxiv.org/abs/2505.09338v1), 2025-05-14

(25) [Statistical Modeling and Uncertainty Estimation of LLM Inference Systems](http://arxiv.org/abs/2505.09319v1), 2025-05-14

(26) [Scent of Knowledge: Optimizing Search-Enhanced Reasoning with Information Foraging](http://arxiv.org/abs/2505.09316v1), 2025-05-14

(27) [Reproducibility Study of "Cooperate or Collapse: Emergence of Sustainable Cooperation in a Society of LLM Agents"](http://arxiv.org/abs/2505.09289v1), 2025-05-14

(28) [A Scalable Unsupervised Framework for multi-aspect labeling of Multilingual and Multi-Domain Review Data](http://arxiv.org/abs/2505.09286v1), 2025-05-14

(29) [A Note on Semantic Diffusion](http://arxiv.org/abs/2505.09283v1), 2025-05-14

(30) [Instantiating Standards: Enabling Standard-Driven Text TTP Extraction with Evolvable Memory](http://arxiv.org/abs/2505.09261v1), 2025-05-14

(31) [Zero-Shot Multi-modal Large Language Model v.s. Supervised Deep Learning: A Comparative Study on CT-Based Intracranial Hemorrhage Subtyping](http://arxiv.org/abs/2505.09252v1), 2025-05-14

(32) [Focus, Merge, Rank: Improved Question Answering Based on Semi-structured Knowledge Bases](http://arxiv.org/abs/2505.09246v1), 2025-05-14

(33) [AMSnet 2.0: A Large AMS Database with AI Segmentation for Net Detection](http://arxiv.org/abs/2505.09155v1), 2025-05-14

(34) [Sensing-Assisted Channel Prediction in Complex Wireless Environments: An LLM-Based Approach](http://arxiv.org/abs/2505.09141v1), 2025-05-14

(35) [ELIS: Efficient LLM Iterative Scheduling System with Response Length Predictor](http://arxiv.org/abs/2505.09142v1), 2025-05-14

(36) [Beyond General Prompts: Automated Prompt Refinement using Contrastive Class Alignment Scores for Disambiguating Objects in Vision-Language Models](http://arxiv.org/abs/2505.09139v1), 2025-05-14

(37) [Air-Ground Collaboration for Language-Specified Missions in Unknown Environments](http://arxiv.org/abs/2505.09108v1), 2025-05-14

(38) [Ornithologist: Towards Trustworthy "Reasoning" about Central Bank Communications](http://arxiv.org/abs/2505.09083v1), 2025-05-14

(39) [CEC-Zero: Chinese Error Correction Solution Based on LLM](http://arxiv.org/abs/2505.09082v1), 2025-05-14

(40) [SALM: A Multi-Agent Framework for Language Model-Driven Social Network Simulation](http://arxiv.org/abs/2505.09081v1), 2025-05-14

(41) [S-DAT: A Multilingual, GenAI-Driven Framework for Automated Divergent Thinking Assessment](http://arxiv.org/abs/2505.09068v1), 2025-05-14

(42) [Variational Prefix Tuning for Diverse and Accurate Code Summarization Using Pre-trained Language Models](http://arxiv.org/abs/2505.09062v1), 2025-05-14

(43) [A Comprehensive Analysis of Large Language Model Outputs: Similarity, Diversity, and Bias](http://arxiv.org/abs/2505.09056v1), 2025-05-14

(44) [Atomic Consistency Preference Optimization for Long-Form Question Answering](http://arxiv.org/abs/2505.09039v1), 2025-05-14

(45) [Improving the Reliability of LLMs: Combining CoT, RAG, Self-Consistency, and Self-Verification](http://arxiv.org/abs/2505.09031v1), 2025-05-13

(46) [Tests as Prompt: A Test-Driven-Development Benchmark for LLM Code Generation](http://arxiv.org/abs/2505.09027v1), 2025-05-13

(47) [Automated Meta Prompt Engineering for Alignment with the Theory of Mind](http://arxiv.org/abs/2505.09024v1), 2025-05-13

(48) [AI-Mediated Code Comment Improvement](http://arxiv.org/abs/2505.09021v1), 2025-05-13

(49) [A suite of LMs comprehend puzzle statements as well as humans](http://arxiv.org/abs/2505.08996v1), 2025-05-13

(50) [ITERA-LLM: Boosting Sub-8-Bit Large Language Model Inference via Iterative Tensor Decomposition](http://arxiv.org/abs/2505.08981v1), 2025-05-13

(51) [Prioritizing Image-Related Tokens Enhances Vision-Language Pre-Training](http://arxiv.org/abs/2505.08971v1), 2025-05-13

(52) [Assessing and Advancing Benchmarks for Evaluating Large Language Models in Software Engineering Tasks](http://arxiv.org/abs/2505.08903v1), 2025-05-13

(53) [Performance Gains of LLMs With Humans in a World of LLMs Versus Humans](http://arxiv.org/abs/2505.08902v1), 2025-05-13

(54) [WaLLM -- Insights from an LLM-Powered Chatbot deployment via WhatsApp](http://arxiv.org/abs/2505.08894v1), 2025-05-13

(55) [Optimized Couplings for Watermarking Large Language Models](http://arxiv.org/abs/2505.08878v1), 2025-05-13

(56) [Generative AI for Autonomous Driving: Frontiers and Opportunities](http://arxiv.org/abs/2505.08854v1), 2025-05-13

(57) [CodePDE: An Inference Framework for LLM-driven PDE Solver Generation](http://arxiv.org/abs/2505.08783v1), 2025-05-13

(58) [HealthBench: Evaluating Large Language Models Towards Improved Human Health](http://arxiv.org/abs/2505.08775v1), 2025-05-13

(59) [SPAT: Sensitivity-based Multihead-attention Pruning on Time Series Forecasting Models](http://arxiv.org/abs/2505.08768v1), 2025-05-13

(60) [Towards Autonomous UAV Visual Object Search in City Space: Benchmark and Agentic Methodology](http://arxiv.org/abs/2505.08765v2), 2025-05-13

(61) [AC-Reason: Towards Theory-Guided Actual Causality Reasoning with Large Language Models](http://arxiv.org/abs/2505.08750v1), 2025-05-13

(62) [DeepMath-Creative: A Benchmark for Evaluating Mathematical Creativity of Large Language Models](http://arxiv.org/abs/2505.08744v1), 2025-05-13

(63) [Probability Consistency in Large Language Models: Theoretical Foundations Meet Empirical Discrepancies](http://arxiv.org/abs/2505.08739v1), 2025-05-13

(64) [NurValues: Real-World Nursing Values Evaluation for Large Language Models in Clinical Context](http://arxiv.org/abs/2505.08734v1), 2025-05-13

(65) [Securing RAG: A Risk Assessment and Mitigation Framework](http://arxiv.org/abs/2505.08728v1), 2025-05-13

(66) [Memorization-Compression Cycles Improve Generalization](http://arxiv.org/abs/2505.08727v1), 2025-05-13

(67) [PWC-MoE: Privacy-Aware Wireless Collaborative Mixture of Experts](http://arxiv.org/abs/2505.08719v1), 2025-05-13

(68) [Improved Algorithms for Differentially Private Language Model Alignment](http://arxiv.org/abs/2505.08849v1), 2025-05-13

(69) [LLM-based Prompt Ensemble for Reliable Medical Entity Recognition from EHRs](http://arxiv.org/abs/2505.08704v1), 2025-05-13

(70) [Granite-speech: open-source speech-aware LLMs with strong English ASR capabilities](http://arxiv.org/abs/2505.08699v2), 2025-05-13

(71) [VizCV: AI-assisted visualization of researchers' publications tracks](http://arxiv.org/abs/2505.08691v1), 2025-05-13

(72) [Adaptive Schema-aware Event Extraction with Retrieval-Augmented Generation](http://arxiv.org/abs/2505.08690v1), 2025-05-13

(73) [A Social Robot with Inner Speech for Dietary Guidance](http://arxiv.org/abs/2505.08664v1), 2025-05-13

(74) [Revealing economic facts: LLMs know more than they say](http://arxiv.org/abs/2505.08662v1), 2025-05-13

(75) [Enhancing Software Development with Context-Aware Conversational Agents: A User Study on Developer Interactions with Chatbots](http://arxiv.org/abs/2505.08648v1), 2025-05-13

(76) [WixQA: A Multi-Dataset Benchmark for Enterprise Retrieval-Augmented Generation](http://arxiv.org/abs/2505.08643v1), 2025-05-13

(77) [TRAIL: Trace Reasoning and Agentic Issue Localization](http://arxiv.org/abs/2505.08638v1), 2025-05-13

(78) [Visually Guided Decoding: Gradient-Free Hard Prompt Inversion with Language Models](http://arxiv.org/abs/2505.08622v1), 2025-05-13

(79) [Resource-Efficient Language Models: Quantization for Fast and Accessible Inference](http://arxiv.org/abs/2505.08620v1), 2025-05-13

(80) [CellTypeAgent: Trustworthy cell type annotation with Large Language Models](http://arxiv.org/abs/2505.08844v1), 2025-05-13

(81) [Automatic Task Detection and Heterogeneous LLM Speculative Decoding](http://arxiv.org/abs/2505.08600v1), 2025-05-13

(82) [Enhancing Thyroid Cytology Diagnosis with RAG-Optimized LLMs and Pa-thology Foundation Models](http://arxiv.org/abs/2505.08590v1), 2025-05-13

(83) [Small but Significant: On the Promise of Small Language Models for Accessible AIED](http://arxiv.org/abs/2505.08588v1), 2025-05-13

(84) [Guiding LLM-based Smart Contract Generation with Finite State Machine](http://arxiv.org/abs/2505.08542v1), 2025-05-13

(85) [The Truth Becomes Clearer Through Debate! Multi-Agent Systems with Large Language Models Unmask Fake News](http://arxiv.org/abs/2505.08532v1), 2025-05-13

(86) [LibVulnWatch: A Deep Assessment Agent System and Leaderboard for Uncovering Hidden Vulnerabilities in Open-Source AI Libraries](http://arxiv.org/abs/2505.08842v1), 2025-05-13

(87) [TrialMatchAI: An End-to-End AI-powered Clinical Trial Recommendation System to Streamline Patient-to-Trial Matching](http://arxiv.org/abs/2505.08508v1), 2025-05-13

(88) [InfoPO: On Mutual Information Maximization for Large Language Model Alignment](http://arxiv.org/abs/2505.08507v1), 2025-05-13

(89) [LCES: Zero-shot Automated Essay Scoring via Pairwise Comparisons Using Large Language Models](http://arxiv.org/abs/2505.08498v1), 2025-05-13

(90) [BizChat: Scaffolding AI-Powered Business Planning for Small Business Owners Across Digital Skill Levels](http://dx.doi.org/10.1145/3707640.3731928), 2025-05-13

(91) [Achieving Scalable Robot Autonomy via neurosymbolic planning using lightweight local LLM](http://arxiv.org/abs/2505.08492v1), 2025-05-13

(92) [Large Language Models Meet Stance Detection: A Survey of Tasks, Methods, Applications, Challenges and Future Directions](http://arxiv.org/abs/2505.08464v1), 2025-05-13

(93) [RepCali: High Efficient Fine-tuning Via Representation Calibration in Latent Space for Pre-trained Language Models](http://arxiv.org/abs/2505.08463v1), 2025-05-13

(94) [Strategy-Augmented Planning for Large Language Models via Opponent Exploitation](http://arxiv.org/abs/2505.08459v1), 2025-05-13

(95) [IterKey: Iterative Keyword Generation with LLMs for Enhanced Retrieval Augmented Generation](http://arxiv.org/abs/2505.08450v1), 2025-05-13

(96) [Scalable UAV Multi-Hop Networking via Multi-Agent Reinforcement Learning with Large Language Models](http://arxiv.org/abs/2505.08448v1), 2025-05-13

(97) [Optimizing Retrieval-Augmented Generation: Analysis of Hyperparameter Impact on Performance and Efficiency](http://arxiv.org/abs/2505.08445v1), 2025-05-13

(98) [A document processing pipeline for the construction of a dataset for topic modeling based on the judgments of the Italian Supreme Court](http://arxiv.org/abs/2505.08439v1), 2025-05-13

(99) [An integrated language-vision foundation model for conversational diagnostics and triaging in primary eye care](http://arxiv.org/abs/2505.08414v1), 2025-05-13

(100) [TUMS: Enhancing Tool-use Abilities of LLMs with Multi-structure Handlers](http://arxiv.org/abs/2505.08402v1), 2025-05-13
